warning  A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event

**Describe the bug** A clear and concise description of what the bug is.

Fix the linting warning by making the interaction for dropping a coin in a column keyboard friendly.

**To Reproduce** Steps to reproduce the behavior:

1. Run `npm run lint`
2. The a11y lint warning appears

```
/Users/nicktaylor/dev/oss/connect-four/src/lib/Board.svelte
  33:3  warning  A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event  a11y-click-events-have-key-events
```

**Expected behavior** A clear and concise description of what you expected to
happen.

The lint warning

```
/Users/nicktaylor/dev/oss/connect-four/src/lib/Board.svelte
  33:3  warning  A11y: visible, non-interactive elements with an on:click event must be accompanied by an on:keydown, on:keyup, or on:keypress event  a11y-click-events-have-key-events
```

should no longer appear and a player is able to use the keyboard to add a coin to a column


**Screenshots** If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**

- OS: [e.g. iOS]
- Browser [e.g. chrome, safari]
- Version [e.g. 22]

**Smartphone (please complete the following information):**

- Device: [e.g. iPhone6]
- OS: [e.g. iOS8.1]
- Browser [e.g. stock browser, safari]
- Version [e.g. 22]

**Additional context** Add any other context about the problem here.

To make this keyboard navigable, the board has to be able to receive focus and tabbing in the board should change columns. Consider focus states as well to indicate which column you're currently on.

--- a/src/lib/Board.svelte
+++ b/src/lib/Board.svelte
@@ -58,6 +58,10 @@
 
 	button[data-column] {
 		cursor: pointer;
+		display: flex;
+		flex-direction: column;
+		justify-content: space-around;
+		/* Reset most button styles */
 		background: transparent none;
 		font-family: inherit;
 		font-size: 100%;
@@ -67,9 +71,6 @@
 		margin: 0;
 		padding: 0;
 		border: 0;
-		display: flex;
-		flex-direction: column;
-		justify-content: space-around;
 	}
 
 	div[data-board] [data-cell] {

--- a/src/lib/Board.svelte
+++ b/src/lib/Board.svelte
@@ -30,7 +30,11 @@
 
 <div data-board>
 	{#each $gameState.boardState as column, colIndex}
-		<button data-column={colIndex} on:click={insertCoinAsCurrentPlayerIntoColumn(colIndex)}>
+		<button
+			data-column={colIndex}
+			on:click={insertCoinAsCurrentPlayerIntoColumn(colIndex)}
+			title={`Insert coin into column ${colIndex + 1}`}
+		>
 			{#each column as cell, rowIndex}
 				<div data-cell="{colIndex},{rowIndex}">
 					{#if cell !== undefined}

--- a/src/lib/Board.svelte
+++ b/src/lib/Board.svelte
@@ -30,7 +30,7 @@
 
 <div data-board>
 	{#each $gameState.boardState as column, colIndex}
-		<div data-column={colIndex} on:click={insertCoinAsCurrentPlayerIntoColumn(colIndex)}>
+		<button data-column={colIndex} on:click={insertCoinAsCurrentPlayerIntoColumn(colIndex)}>
 			{#each column as cell, rowIndex}
 				<div data-cell="{colIndex},{rowIndex}">
 					{#if cell !== undefined}
@@ -40,7 +40,7 @@
 					{/if}
 				</div>
 			{/each}
-		</div>
+		</button>
 	{/each}
 </div>
 
@@ -54,6 +54,22 @@
 		width: 28rem;
 		height: 25rem;
 		background-color: blue;
+	}
+
+	button[data-column] {
+		cursor: pointer;
+		background: transparent none;
+		font-family: inherit;
+		font-size: 100%;
+		font-weight: inherit;
+		line-height: inherit;
+		color: inherit;
+		margin: 0;
+		padding: 0;
+		border: 0;
+		display: flex;
+		flex-direction: column;
+		justify-content: space-around;
 	}
 
 	div[data-board] [data-cell] {

